docs/source/tutorials/audio.ipynb

```diff
@@ -19,6 +19,56 @@
     "- Use cleanlab to identify a list of audio clips with potential label errors.\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "Quickstart\n",
+    "<br/>\n",
+    "    \n",
+    "Already have an sklearn compatible `model`, audio `data` and true `labels`? Run the code below to train your `model` and get label issues.\n",
+    "\n",
+    "\n",
+    "<div  class=markdown markdown=\"1\" style=\"background:white;margin:16px\">  \n",
+    "    \n",
+    "```python\n",
+    "\n",
+    "from cleanlab.classification import CleanLearning\n",
+    "\n",
+    "cl = CleanLearning(model)\n",
+    "_ = cl.fit(train_data, labels)\n",
+    "label_issues = cl.get_label_issues()\n",
+    "preds = cl.predict(test_data) # predictions from a version of your model \n",
+    "                              # trained on auto-cleaned data\n",
+    "\n",
+    "\n",
+    "```\n",
+    "    \n",
+    "</div>\n",
+    "    \n",
+    "Is your model/data not compatible with `CleanLearning`? You can instead run cross-validation on your model to get out-of-sample `pred_probs` on true `labels`. Then run the code below to get label issue indices ranked by their inferred severity.\n",
```

Comment:
Let's delete this and only demonstrate `filter.find_label_issues()` in the quickstart.  This tutorial never mentions CleanLearning, and I think it's confusing to mention CleanLearning in this blob when it is never illustrated anywhere in the tutorial.  In general, these quickstart blobs should only cover functionality later illustrated in the tutorial.